src/hip/hip_build_utils.cpp

```diff
@@ -39,6 +39,44 @@
 MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_DEBUG_HIP_VERBOSE)
 MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_DEBUG_HIP_DUMP)
 
+#ifdef MIOPEN_COMPILER_FLAGS_V2
+  /* List to hold Relative Directory Path
+   * for each Compiler Flags from ROCM Install Path
+   * The definition of this variable expected to be available in C/C++
+   * code of the Target Library */
+  char* MIOPEN_CMP_FLAG_REL_PATHS[MIOPEN_CMP_MAX_Flag] = { "llvm/bin/clang",
+                                                         "bin/clang-ocl",
+                                                         "llvm/bin/clang++",
```

Comment:
shouldn't it be `"bin/amdclang++"` ? @junliume @JehandadKhan

Reply:
@atamazov There should be no behavior differences between `/opt/rocm/bin/amdclang++` and `/opt/rocm/llvm/bin/clang++` ; `amdclang++` was added to name-differentiate our clang from other clangs that may be installed on a system.

Reply:
[resolved]